#!/usr/bin/env python
PACKAGE = "yumi_motion"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# NAME			TYPE	  RECONFIGURATION LEVEL	    DESCRIPTION               DEFAULT  	   MIN   MAX
gen.add("left_cfg",    bool_t,             0,                  "start", 		False)
gen.add("right_cfg",   bool_t,             0,                  "start", 		False)

 
exit(gen.generate(PACKAGE, "yumi_motion", "yumimotion"))
